security-common/src/main/kotlin/org/octopusden/cloud/commons/security/client/dto/OfflineJwt.kt

```
@@ -5,25 +5,25 @@ import java.time.Instant
 
 data class OfflineJwt(
     @JsonProperty("access_token") val accessToken: String,
-    @JsonProperty("refresh_token") val refreshToken: String
+    @JsonProperty("refresh_token") val refreshToken: String,
 ) {
     lateinit var refreshTokenExpDate: Instant
```

detekt (1.23.8)

[warning] Usages of lateinit should be avoided.

(detekt.potential-bugs.LateinitUsage)